Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace staff
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {   
+            List<dynamic> staffList = new List<dynamic>();
+            int empId = 1;
+            while (true)
+            {   
+                MenuActinos menuActions = new MenuActinos();
+                Console.WriteLine("\nSelect the Action \n1.Add a staff\n2.view a staff\n3.view all staff\n4.Update a staff info\n5.Delete a Staff");
+                int userChoice = Convert.ToInt32(Console.ReadLine());
+                switch(userChoice) 
+                {
+                case 1:
```

Comment:
Isn't our requirement states that the main menu is to manage staff followed by the operations?

Reply:
Seems like it is reversed here